src/cli/index.js

```diff
@@ -124,6 +124,7 @@ export function runCli(info = { version: 'Unknown', name: 'Unknown' }, initalCon
         // Run the command
         return configObject.commands[command]({
             verbose: verboseMode,
+            directory: dirPath,
```

Comment:
I like this change and has been on my list of things to address. Could we add a comment about this in the documentation? https://github.com/rocjs/roc/blob/master/docs/config/commands.md#function-command